REQUIRE

```diff
@@ -1 +1 @@
-julia 0.6-
+julia 0.6.0-pre.beta.450
```

Comment:
@tkelman Is this format correct? I took the merge commit and used `contrib/commit-name.sh`